src/core/common/linked_list.hpp

```diff
@@ -513,10 +516,10 @@ template <typename Type> class LinkedList
      *
      * @param[in]  aIndicator  An indicator to match with entries in the list.
      * @param[out] aPrevEntry  A pointer to output the previous entry on success (when a match is found in the list).
-     *                         @p aPrevEntry is set to nullptr if the matching entry is the head of the list. Otherwise
-     *                         it is updated to point to the previous entry before the matching entry in the list.
+     *                         @p aPrevEntry is set to `nullptr` if the matching entry is the head of the list.
+     * Otherwise it is updated to point to the previous entry before the matching entry in the list.
```

Comment:
```suggestion
     *                         @p aPrevEntry is set to `nullptr` if the matching entry is the head of the list.
     *                         Otherwise it is updated to point to the previous entry before the matching entry in the
     *                         list.
```